pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper_test.go

```diff
@@ -19,3 +25,258 @@ func TestNewGitHubScraper(t *testing.T) {
 
 	assert.NotNil(t, s)
 }
+
+func TestGetTotalPrPages(t *testing.T) {
```

Comment:
When changing to `getNumPRPages` you'll have to go back through and change these tests.